src/test/java/org/jenkinsci/plugins/workflow/steps/CatchErrorStepTest.java

```diff
@@ -162,26 +162,26 @@ public class CatchErrorStepTest {
         assertCatchError(r, b, Result.SUCCESS, Result.UNSTABLE, true);
     }
 
-    @Test public void catchesTimeoutsByDefault() throws Exception {
+    @Test public void catchesInterruptionsByDefault() throws Exception {
         WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
         p.setDefinition(new CpsFlowDefinition(
-                "timeout(time: 1, unit: 'SECONDS') {\n" +
-                "  catchError(message: 'caught error') {\n" +
-                "    sleep 5\n" +
-                "  }\n" +
-                "}", true));
+                "import jenkins.model.CauseOfInterruption\n" +
+                        "import org.jenkinsci.plugins.workflow.steps.FlowInterruptedException\n" +
+                        "catchError(message: 'caught error') {\n" +
+                        "  throw new FlowInterruptedException(Result.ABORTED, true, new CauseOfInterruption[0])\n" +
+                        "}", false));
```

Comment:
I would align the indentation with the first line to make it easier to read the script.

```suggestion
                "import org.jenkinsci.plugins.workflow.steps.FlowInterruptedException\n" +
                "catchError(message: 'caught error') {\n" +
                "  throw new FlowInterruptedException(Result.ABORTED, true, new CauseOfInterruption[0])\n" +
                "}", false));
```